modules/app-all/main.tf

```diff
@@ -113,7 +113,7 @@ module "anaml-server" {
   kubernetes_security_context = var.override_anaml_server_kubernetes_security_context
 
   kubernetes_service_account_name = coalesce(
-    var.override_anaml_spark_server_kubernetes_service_account,
+    var.override_anaml_server_kubernetes_service_account,
```

Comment:
I can't remember why I did this but looks like I got `anaml_spark_server` and `anaml_server` mixed up. This block is `anaml_server`, the one below is the `anaml_spark_server` config.